Documentation/network/standalone.rst

```diff
@@ -0,0 +1,1610 @@
+
+.. note::
+
+  The above step is only required for distributions with older systemd versions.
+  For most modern distributions this step is not required as systemd will
+  automatically mount the BPF file system at its default location. See also
+  `GH-10955 <https://github.com/cilium/cilium/issues/10955>`_ for details.
+
+In this Quick-Start example, we enable the standalone layer 4 load-balancer
+functionality for both IPv4 and IPv6, with Maglev consistent hashing as the
+load-balancing algorithm. In our case, the networking device for the load-balancer
+is called ``bond0``. Multi-device setups are supported as well, and can be
+specified via ``--devices=bond0,eth0`` or through a device name regex.
+
+For the specified devices, we enable XDP acceleration for maximum efficiency.
+The vast majority of modern network drivers for 10/40/100G and beyond support
+XDP today. If you are unsure whether your network driver supports native XDP,
+then simply set ``--bpf-lb-acceleration=disabled`` instead and Cilium loads
+its programs into the tc BPF layer instead of XDP.
+
+.. parsed-literal::
+
+   docker run --name cilium-gateway -td \\
+     -v /sys/fs/bpf:/sys/fs/bpf \\
+     -v /lib/modules:/lib/modules \\
+     --privileged=true \\
+     --network=host \\
+     quay.io/cilium/cilium:|IMAGE_TAG| \\
+     cilium-agent \\
+     --enable-ipv4=true \\
+     --enable-ipv6=true \\
+     --datapath-mode=lb-only \\
+     --bpf-lb-algorithm=maglev \\
+     --bpf-lb-acceleration=native \\
+     --devices=bond0
+
+By default, the layer 4 load-balancer operates in Source NAT (SNAT) mode, meaning replies from
+backends will be returned to the load-balancer which will then perform reverse NAT and
+sends the reply back to the client. Advanced options such as Direct Server Return
+(DSR) are supported as well and described in later sections of this guide.
+
+Validate the Setup
+==================
+
+After deploying Cilium Standalone Gateway with the Quick-Start guide above, we can first
+validate that the Cilium agent is running in the desired mode:
+
+.. parsed-literal::
+
+   docker exec cilium-gateway cilium status --verbose
+   [...]
+   Kubernetes:             Disabled
+   KubeProxyReplacement Details:
+     Status:                 Partial
+     Socket LB:              Enabled
+     Socket LB Protocols:    TCP, UDP
+     Devices:                bond0 86.109.5.207 2604:1380:4091:cf00::1 (Direct Routing)
+     Mode:                   SNAT
+     Backend Selection:      Maglev (Table Size: 16381)
+     Session Affinity:       Disabled
+     Graceful Termination:   Enabled
+     NAT46/64 Support:
+     - Services:             Enabled
+     - Gateway:              Disabled
+     XDP Acceleration:       Native
+   [...]
+
+As an optional next step, we will create a simple service entry for testing purpose.
+Then we will validate that Cilium installed the service correctly.
+
+Cilium exposes its programmable API under ``unix:///var/run/cilium/cilium.sock``.
```

Comment:
We probably want to drop a note here somewhere that the API is subject to change? I don't think we're currently enforcing API stability on these changes.